src/torchaudio/models/decoder/__init__.py

```diff
@@ -36,7 +36,10 @@ def __getattr__(name: str):
             ) from err
 
         orig_item = getattr(_cuda_ctc_decoder, name)
-        if inspect.isclass(orig_item):
+        if (
+                inspect.isclass(orig_item)
+                or (getattr(orig_item, "__sphinx_mock__", False) and inspect.isclass(orig_item.__class__))
+        ):
```

Comment:
Can you share more about why this was needed? In general we should avoid having doc-related (or test-related) logic within the `src/` folder.

Reply:
oh boy, OK. Thanks for the explanations. Let's merge, knowing that hopefully we'll be removing the call to `dropping_*` soon. Before merging, do you mind adding a comment to that effect, indicating that the `new `if` block should be removed?

Reply:
Done adding comments.